Makefile

```diff
@@ -313,6 +314,9 @@ providers/build/aws: providers/lr
 providers/install/aws:
 	@$(call installProvider, providers/aws)
 
+providers/build/atlassian: providers/lr
```

Comment:
can you add the `providers/install/atlassian` target too

Reply:
It's there